src/collection.ts

```diff
@@ -34,6 +32,7 @@ export default class Collection {
   public client: KintoClientBase;
   private bucket: Bucket;
   public name: string;
+  private _endpoints: any;
```

Comment:
Same as above: let's try to avoid using `any`. You _should_ be able to use `KintoClientBase["endpoints"]` here.